zeebe/broker/src/main/java/io/camunda/zeebe/broker/system/partitions/impl/AsyncSnapshotDirector.java

```diff
@@ -378,9 +382,17 @@ private void resetStateOnFailure(final Throwable failure) {
     ongoingSnapshotFuture = null;
   }
 
+  @Override
+  public ActorFuture<PersistedSnapshot> takeSnapshot(
+      final int partition, final long lastProcessedPosition, final ConcurrencyControl control) {
```

Comment:
:x: The parameter ConcurrencyControl is unused. You can remove it I think.

Reply:
Done :+1: